README.md

```diff
@@ -1,72 +1,69 @@
 # Circuits for zkEVM
 
-This is the zkEVM circuits used in Scroll Mainnet. It was forked from PSE zkevm-circuits and added a lot of new features later:
+This is the zkEVM circuits used in Morph Mainnet. It was forked from Scroll zkevm-circuits and a lot of new features were added later and in the future:
 
-1. SHA256 / MODEXP / EC precompiles
-2. RLP circuit / MPT circuit / Poseidon circuit
-3. multi block chunking
-4. proof aggregation
-5. [>99.5% compatiblity](https://circuit-release.s3.us-west-2.amazonaws.com/testool/nightly.1695216104.47e2015.html) with [official EVM test vector](https://github.com/ethereum/tests)
-6. Many optimizations like read/write memory in word instead of byte
+1. Sequencer set circuit for decentralized sequencer
+2. EIP4844 circuit
+3. Faster hash function circuits for zktrie
```

Comment:
Optimal zk-friendly permutation schemes used in the hash function for zktrie, along with the circuits.